xcube/util/cmaps.py

```diff
@@ -477,11 +495,11 @@ def parse_cm_code(cm_code: str) -> tuple[str, Optional[Colormap]]:
         cm_base_name, _, _ = parse_cm_name(cm_name)
         n = len(cm_items)
         if cm_type == "categorical":
-            values: list[int] = []
+            values: list[float] = []
             colors: list[Union[str, tuple[float, ...]]] = []
             bad = 0, 0, 0, 0
             for i, (value, color) in enumerate(cm_items):
-                key = int(value)
+                key = float(value)
```

Comment:
Please revert, see above.

Reply:
see above

Reply:
I cannot believe that "Object of type int64 is not JSON serializable" results from `key = int(value)`. I will check.

Reply:
`json.dumps(int(3))` works, and even `json.dumps(int(3000000000000000000000000000000))`.

Reply:
This will produce the above error:

Reply:
Yes, please find out why at all our `int` values are turned into a numpy array.

Reply:
Found it. My bad. I applied `np.min` and `np.max to the values of the cmap. I changed that, and it works now with integers.

Reply:
Great!